package.json

```diff
@@ -23,14 +24,19 @@
     "bent": "^7.0.2",
     "bull": "^3.11.0",
     "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "express-healthcheck": "^0.1.0",
     "feedparser-promised": "^2.0.1",
+    "minimist": "^1.2.0",
```

Comment:
Let's remove `minimist` and `opn`.  The latter isn't used at all, and the former isn't needed, since we won't be starting the server directly from the command line (see comments below).